src/scss/components/_listings.scss

```diff
@@ -40,9 +41,19 @@ $listing-promoIconColor         : $orange;
     }
 }
 
+.c-listing--preorder,
 .c-listing--offline {
-    padding: spacing(x2);
-    background: $listing-offlineBackground;
+    margin-top: spacing(x3);
+}
+
+.c-listing--preorder {
+    overflow: hidden;
+    padding: 0 spacing(x2) spacing(x2);
+    background: $listing-preorderBackground;
+
+    &:first-child {
+        background: red !important;;
```

Comment:
Should reference a variable for colours?

Reply:
this was me doing a stupid 🤦‍♂️  _deletes_